### PR TITLE
Add integration test for option optimize for large inputs #106

### PR DESCRIPTION
**Pull request by allieychen**
- Create a folder in `gcp-variant-transforms-testfiles/large_tests` with 70000 copies of file `valid_4_2` to test `LARGE_DATA_THRESHOLD = 50000` pipeline.
- Option `--optimize_for_large_inputs` is included.

Tested: Manually ran deploy_and_run_tests.sh --run_presubmit_tests
Issue: #106

### CODE REVIEW

File: gcp_variant_transforms/testing/integration/presubmit_tests/large_tests/option_optimize_for_large_inputs.json [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+        "SELECT COUNT(0) AS num_rows FROM ( ",
+        "SELECT reference_name, start_position, end_position ",
+        "FROM  {TABLE_NAME} GROUP BY 1, 2, 3)"
+      ],
+      "expected_result": {"num_rows": 13}
+    },
+    {
+      "query": [
+        "SELECT COUNT(0) AS num_rows FROM {TABLE_NAME} AS t, t.call AS call ",
+        "WHERE start_position = 17329"
```
Comment (arostamianfar) [inline gcp_variant_transforms/testing/integration/presubmit_tests/large_tests/option_optimize_for_large_inputs.json]: I think this is a bit confusing as it's not obvious that there are two chromosomes with start_position 17329 (and verification of why there are 6 calls is difficult). Consider just using '20' or 'Y' here (the aggregation logic is tested in the query below anyway).

File: gcp_variant_transforms/testing/integration/presubmit_tests/large_tests/option_optimize_for_large_inputs.json [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+      "query": [
+        "SELECT COUNT(0) AS num_rows FROM ( ",
+        "SELECT reference_name, start_position, end_position ",
+        "FROM  {TABLE_NAME} GROUP BY 1, 2, 3)"
+      ],
+      "expected_result": {"num_rows": 13}
+    },
+    {
+      "query": [
+        "SELECT COUNT(0) AS num_rows FROM {TABLE_NAME} AS t, t.call AS call ",
```
Comment (arostamianfar) [inline gcp_variant_transforms/testing/integration/presubmit_tests/large_tests/option_optimize_for_large_inputs.json]: s/num_rows/num_calls

File: gcp_variant_transforms/testing/integration/presubmit_tests/large_tests/option_optimize_for_large_inputs.json [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+      ],
+      "expected_result": {"num_rows": 420000}
+    },
+    {
+      "query": [
+        "SELECT COUNT(0) AS num_rows FROM {TABLE_NAME} AS t, t.call AS call "
+      ],
+      "expected_result": {"num_rows": 2730000}
+    }
+  ]
```
Comment (arostamianfar) [inline gcp_variant_transforms/testing/integration/presubmit_tests/large_tests/option_optimize_for_large_inputs.json]: Please add this query as well to ensure the actual genotypes are still correct (I couldn't find a way to return all genotypes in one query, but this is ok, you may optionally create another query to count genotype > 0 as well):
```
SELECT
  COUNT(*) num_ref_calls
FROM {TABLE_NAME} AS t, t.call AS call, call.genotype AS genotype
WHERE genotype = 0
GROUP BY genotype
```

File: gcp_variant_transforms/testing/integration/presubmit_tests/large_tests/option_optimize_for_large_inputs.json [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+    {
+      "query": [
+        "SELECT COUNT(0) AS num_rows FROM {TABLE_NAME} AS t, t.call AS call ",
+        "WHERE start_position = 17329"
+      ],
+      "expected_result": {"num_rows": 420000}
+    },
+    {
+      "query": [
+        "SELECT COUNT(0) AS num_rows FROM {TABLE_NAME} AS t, t.call AS call "
```
Comment (arostamianfar) [inline gcp_variant_transforms/testing/integration/presubmit_tests/large_tests/option_optimize_for_large_inputs.json]: s/num_rows/num_calls

File: gcp_variant_transforms/testing/integration/presubmit_tests/large_tests/option_optimize_for_large_inputs.json (hunk 2 above)
Comment (allieychen) [reply gcp_variant_transforms/testing/integration/presubmit_tests/large_tests/option_optimize_for_large_inputs.json]: Done.

File: gcp_variant_transforms/testing/integration/presubmit_tests/large_tests/option_optimize_for_large_inputs.json (hunk 4 above)
Comment (allieychen) [reply gcp_variant_transforms/testing/integration/presubmit_tests/large_tests/option_optimize_for_large_inputs.json]: Done.

File: gcp_variant_transforms/testing/integration/presubmit_tests/large_tests/option_optimize_for_large_inputs.json (hunk 1 above)
Comment (allieychen) [reply gcp_variant_transforms/testing/integration/presubmit_tests/large_tests/option_optimize_for_large_inputs.json]: Good point! I am going to modify `test_1000_copies_of_valid_4_2_with_merge` as well.

File: gcp_variant_transforms/testing/integration/presubmit_tests/large_tests/option_optimize_for_large_inputs.json (hunk 3 above)
Comment (allieychen) [reply gcp_variant_transforms/testing/integration/presubmit_tests/large_tests/option_optimize_for_large_inputs.json]: Thanks. Done.

**Comment by allieychen** [comment]
PTAL. :)

**Pull request merged**